libcodechecker/server/api/report_server.py

```diff
@@ -410,6 +410,18 @@ def get_report_path_hash(report, files):
     return hashlib.md5(report_path_hash.encode()).hexdigest()
 
 
+def filter_outstanding_reports(q):
+    skip_review_status = list(map(review_status_str, [
+        ttypes.ReviewStatus.FALSE_POSITIVE,
+        ttypes.ReviewStatus.INTENTIONAL]))
```

Comment:
Don't we already have a function that returns these review statuses that we want to skip?